loader/src/sources/cdc/api.js

```diff
@@ -324,18 +333,25 @@ const mislabeledLocations = {
   "019de7c9-956f-40b5-8e3b-deb67c8a1b95": { system: "walmart", value: "6690" },
 };
 
-function getStoreExternalId(location) {
+function getStoreExternalIds(location) {
   const correction = mislabeledLocations[location.provider_location_guid];
   if (correction) {
-    return [correction.system, correction.value];
+    return [[correction.system, correction.value]];
   }
 
   for (const definition of locationSystems) {
     if (definition.pattern.test(location.loc_name)) {
       const idValue = (definition.getId || getSimpleId)(location);
-      return idValue ? [definition.system, idValue] : null;
+      if (idValue) {
+        return Array.isArray(idValue)
+          ? idValue
+          : [[definition.system, idValue]];
+      }
+      return [];
```

Comment:
Do you need this line, or can you safely fall through to the similar `return []` at the end of the function?

Reply:
Ugh, the revision here probably made things too subtle: it can’t fall through because it needs to stop iterating once it’s found a matching entry in `locationSystems`. This matches the existing behavior, where an empty result from the location system causes the location to be dropped from the output. (OTOH, I can see a version of the universe where that should be an error we get alerted about, but… that seems like a big change to make here.)

Reply:
Realized I could switch this to a `break`, which is a *little* clearer, and means we don’t repeat `return []` in multiple places.